vgen/src/main/java/io/gapi/vgen/php/PhpContextCommon.java

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gapi.vgen.php;
+
+import com.google.common.collect.BiMap;
+import com.google.common.collect.HashBiMap;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class PhpContextCommon {
+
+  /**
+   * A bi-map from full names to short names indicating the import map.
+   */
+  private final BiMap<String, String> imports = HashBiMap.create();
+
+  public String getMinimallyQualifiedName(String fullName, String shortName) {
```

Comment:
This method does two things:
(1) find the minimally qualified name -> I can tell this from the method name
(2) add the full name -> short name to the imports map -> I cannot tell this from the method name since it has the prefix 'get'

I suggest to have a separate method to handle (2)